src/put.js

```diff
@@ -59,6 +61,34 @@ module.exports = {
           res.status(500).send(err.message)
         })
     })
+
+    putMetaHandler = (context, path, value, cb) => {
+      let defaults
+
+      try {
+        defaults = skConfig.readDefaultsFile(app)
+      } catch (e) {
+        defaults = {
+          vessels: {
+            self: {}
+          }
+        }
+      }
+      const full = context + '.' + path
+      _.set(defaults, full, value)
+      _.set(app.config.defaults, full, value)
```

Comment:
If defaults data is in memory and we need to change that why do we read the defaults from the file? Or in reverse: why are we modifying defaults data read from the file and `app.config.defaults` separately and not just setting `app.config.defaults = defaults`?

Reply:
good point. will fix...

Reply:
Cleaned that stuff up, please take a look.